assets/src/edit-story/components/panels/design/animation/animation.js

```diff
@@ -207,24 +222,26 @@ function AnimationPanel({
     </SimplePanel>
   ) : (
     <SimplePanel name="animation" title={__('Animation', 'web-stories')}>
```

Comment:
Does this panel have the proper bottom padding? Looks a bit cramped here between the animation input and the layer panel (this is scrolled all the way to the bottom):

![Screen Shot 2021-03-19 at 11 29 55](https://user-images.githubusercontent.com/637548/111804796-ca01db00-88a6-11eb-9bff-c9aefa5a130c.png)

Reply:
Good catch, it did not have it. Thanks!